lib/sim.c

```diff
@@ -562,6 +569,7 @@ sim_add_pixel_to_occupancy(sim_t *self, unsigned int h, unsigned int pixel)
             goto out;
         }
         if (avl_insert_node(tree, h_node) == NULL) {
+            printf("Insert node error");
```

Comment:
Are these printfs necessary? We can keep them for now if they're useful for debugging, but they should be cleared out once we're happy everything's working OK.